package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "dev": "next",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "test": "jest --watch",
+    "test:ci": "jest --ci"
```

Comment:
Seems that you have to add the `--passWithNoTests` flag to make the test command exit with a 0 code so that the Github check passes. Otherwise the changes look great!

Reply:
I just updated the flag for it 👌